modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/logit/storage/file/segment/SegmentFile.java

```diff
@@ -102,9 +102,9 @@ public byte[] lookupData(final long logIndex, final int pos) {
                     getFilePath(), logIndex, pos, this.header.getFirstLogIndex(), getLastLogIndex());
                 return null;
             }
-            if (pos > getFlushedPosition()) {
+            if (pos > getWrotePosition()) {
```

Comment:
I don't understand this change, could you please elaborate?

Reply:
I forgot to comment on it, I apologize. It is a bug fix, to my knowledge